src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBannerCircular.vue

```diff
@@ -344,6 +345,7 @@ export default {
 
 	&__progress-circle,
 	&__image {
+		display: flex;
```

Comment:
I believe we can remove this display:flex, if the image width and height are set correctly on line 29. I was seeing a width=164 and height=164 should work.

Reply:
Updated in the latest commit, thanks for flagging this @ryan-ludwig.